crmsh/bootstrap.py

```diff
@@ -1837,6 +1869,10 @@ def init_qdevice():
         utils.disable_service("corosync-qdevice.service")
         return
 
+    if _context.stage == "qdevice":
+        utils.check_all_nodes_reachable()
+        _context.qdevice_reload_policy = evaluate_qdevice_quorum_effect("add")
```

Comment:
I would replace this `add` by a constant

Reply:
Changed